doc/source/api/io.rst

```diff
@@ -0,0 +1,50 @@
+File-IO
```

Comment:
Could we change this to "Cuds-File", or something else.  This might be a dumb nitpick that can be ignored but was just wondering if we could avoid saying the term FILE-IO to avoid confusion with 'FILE-IO wrapper'